descartes_light/core/include/descartes_light/solvers/ladder_graph/impl/ladder_graph_solver.hpp

```diff
@@ -130,6 +131,18 @@ BuildStatus LadderGraphSolver<FloatType>::buildImpl(
   double duration = std::chrono::duration<double>(Clock::now() - start_time).count();
   CONSOLE_BRIDGE_logDebug("Descartes took %0.4f seconds to build vertices.", duration);
 
+  if (!status.failed_vertices.empty())
+  {
+    CONSOLE_BRIDGE_logDebug("Failed vertices and positions: ");
+    for (auto&& vert_idx : status.failed_vertices)
+    {
+      auto pos = trajectory[vert_idx];
+      std::stringstream s;
+      s << vert_idx << ": " << *pos;
+
+      CONSOLE_BRIDGE_logDebug("%s", s.str().c_str());
```

Comment:
```suggestion
    std::stringstream s;
    s << "Failed vertices:\n";
    for (auto&& vert_idx : status.failed_vertices)
    {
      s << vert_idx << ": " << *trajectory[vert_idx] << "\n";
    }
    CONSOLE_BRIDGE_logDebug(ss.str().c_str());
```

Reply:
In my experience sometimes doing it like this cuts off part of the string. I don't know if it's a ROS or terminal thing.